L/LibCURL/common.jl

```diff
@@ -39,6 +39,11 @@ function build_libcurl(ARGS, name::String)
         --enable-versioned-symbols
     )
 
+    if [[ ${bb_full_target} == *-sanitize+memory* ]]; then
+    # Install msan runtime (for clang)
+        cp -rL ${prefix}/lib/linux/* /opt/x86_64-linux-musl/lib/clang/13.0.1/lib/linux/
```

Comment:
To avoid having to change this line when we update the toolchain:
```suggestion
        cp -rL ${libdir}/linux/* /opt/x86_64-linux-musl/lib/clang/*/lib/linux/
```